### DISCUSSION TITLE
HELP not detecting anything on my set of images

### DISCUSSION THREAD

**Discussion by vladyskai** [mindee/doctr#1286] [Q&A]
I'm not sure why but it is not detecting anything on images like the one attached. If I run it through Google Cloud Vision it does return the correct text, so I believe something is up with the model. I did run other files and images and it works perfectly, so my installation is correct. 

![test_image1](https://github.com/mindee/doctr/assets/116801803/d2ec5606-9044-48d8-a873-88f82462cbf8)

I am using the basic code:

```
from doctr.io import DocumentFile
from doctr.models import ocr_predictor

model = ocr_predictor(pretrained=True)

single_img_doc = DocumentFile.from_images(r"C:\Users\0272455\OCR_doctr_test\test_image1.jpg")

# Analyze
result = model(single_img_doc)

result.show(single_img_doc)
```

**Comment by felixdittrich92** [comment]
Hi @vladyskai 👋 ,

Please try to pass `preserve_aspect_ratio=True` to the `ocr_predictor` .

The detection models are pretrained on different document types and resized / padded to a size of 1024x1024 (which is the default for our detection models). So the problem is that this image seems to be to small.

**Reply by vladyskai** [reply]
Thank you for the answer!

Did it, but unfortunately there are still no detections. I guess the only solution is to train it on smaller images :(

**Reply by felixT2K** [reply]
Unfortunately yes, or you try to upscale the image with https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/upscale for example.
